package/MDAnalysis/analysis/dielectric.py

```diff
@@ -117,14 +118,14 @@ class DielectricConstant(AnalysisBase):
 
     .. versionadded:: 2.1.0
     """
-    def __init__(self, atomgroup, temperature=300, make_whole=True, **kwargs):
+    def __init__(self, atomgroup: AtomGroup, temperature: float = 300, make_whole: bool = True, **kwargs: int) -> None:
```

Comment:
```suggestion
    def __init__(self, atomgroup: AtomGroup, temperature: float = 300, make_whole: bool = True, **kwargs) -> None:
```

The keyword arguments are not ints. They can be anything that AnalysisBase will accept.